File: Programmers/C++/Level 1/크기가 작은 부분문자열/solution.cpp
// https://school.programmers.co.kr/learn/courses/30/lessons/147355?language=cpp#


#include <string>
#include <vector>

using namespace std;

int solution(string t, string p) {
    int count = 0;
    int left = 0;
    for (int i = 0; i <= t.size() - p.size(); i++){
        int right = i + p.size();
        if (stol(t.substr(i, p.size())) <= stol(p)){
            count += 1;
        }
    }
    return count;
}

File: Programmers/Python/Level 1/크기가 작은 부분문자열/solution.py
# https://school.programmers.co.kr/learn/courses/30/lessons/147355


def solution(t, p):
    count = 0
    for left in range(len(t) - len(p) + 1):
        right = left + len(p)
        if int(t[left:right]) <= int(p):
            count += 1
    return count
